code/game/objects/items/devices/modificationkit.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/modkit
+	name = "modification kit"
+	desc = "A one-use kit, which enables kinetic accelerators to be fired with only one hand, as well as allowing less dextrous races to use the tool."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "modkit"
+	origin_tech = "programming=2;materials=2;magnets=4"
+	var/uses = 1
+
+/obj/item/modkit/afterattack(var/obj/item/weapon/gun/energy/kinetic_accelerator/C, mob/user)
```

Comment:
I mean technically you could PR to this branch like coiax did, but this works as it is right now and I recall it not working at all until oranges helped me and told me to put the var there.